camelot/view/action_steps/change_object.py

```diff
@@ -287,6 +287,10 @@ class ChangeObject(OpenFormView):
     reject: typing.Union[str, ugettext_lazy] = _('Cancel')
     blocking: bool = True
 
+    def __post_init__(self, value, admin, proxy):
+        super().__post_init__(value, admin, proxy)
+        self.title = admin.get_verbose_name()
```

Comment:
dit gaat denk ik in 99% vd gevallen wel werken, maar het echte probleem is dat str() op een dataclass w gecalled, waar de __str__ niet overschreven is en daardoor de hele inhoud van die dataclass 'gedumped' w in de str ...